utils/neetcode.py

```diff
@@ -0,0 +1,28 @@
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    # To prevent circular imports
+    from bot import DiscordBot
+from constants import Language
+
+async def generate_neetcode_link(bot: "DiscordBot", question_id: str, question_title: str, language: Language):
```

Comment:
There is no need for this function to be a coroutine, so remove the `async` from this function, and the `await`s used when calling this function.